cmd/migrate.go

```diff
@@ -0,0 +1,95 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/manifoldco/promptui"
+	"github.com/olekukonko/tablewriter"
+	"github.com/kubernetes-sigs/pspmigrator"
+	"github.com/spf13/cobra"
+	v1 "k8s.io/api/core/v1"
+	psaApi "k8s.io/pod-security-admission/api"
+)
+
+var MigrateCmd = &cobra.Command{
+	Use:   "migrate",
+	Short: "Interactive command to migrate from PSP to PSA ",
+	Long: `The interactive command will help with setting a suggested a
+	Suggested Pod Security Standard for each namespace. In addition, it also
+	checks whether a PSP object is mutating pods in every namespace.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		pods := GetPods()
+		fmt.Println("Checking if any pods are being mutated by a PSP object")
+		mutatedPods := make([]v1.Pod, 0)
+		for _, pod := range pods.Items {
+			mutated, _, err := pspmigrator.IsPodBeingMutatedByPSP(&pod, clientset)
+			if err != nil {
+				log.Println(err)
```

Comment:
If an error occurred in this step, it's probably safest to abort, since there may be un-evaluated pods.
```suggestion
				log.Fatal(err)
```

Reply:
hmm I was thinking it's preferable not to abort so all pods with errors are shown. I guess it depends on how impactful the error might be. Agree safest would be to abort, I will change that in the updated PR.

Reply:
For example this error is aborting the whole flow whereas it is safe to continue:
```
2022/07/21 13:51:14 replicasets.apps "pspmigrator-mutating-podspec-554489987f" not found
```

Reply:
I did end up changing to Fatalln. Done.